clang/lib/CodeGen/CGDebugInfo.h

```diff
@@ -659,8 +659,11 @@ class CGDebugInfo {
   ///
   /// This is used to indiciate instructions that come from compiler
   /// instrumentation.
-  llvm::DILocation *CreateSyntheticInlineAt(llvm::DebugLoc Location,
-                                            StringRef FuncName);
+  llvm::DILocation *CreateSyntheticInlineAt(llvm::DebugLoc ParentLocation,
+                                            llvm::DISubprogram *SynthSP);
```

Comment:
Nit: Is it worth abbreviating `Subprogram` to `SP` here? I saw that and I was first thinking "shared pointer" (a convention that LLDB uses) and then realized it probably meant `Subprogram`.